client/components/code_editor/code-editor-controller.js

```diff
@@ -123,6 +123,17 @@ explorer.components.code_editor.CodeEditorCtrl = function(
   };
 
   /**
+   * Options for the CodeMirror editor.
+   * @export {!Object}
+   */
+  this.editorOptionsSQLReadOnly = {
+    lineWrapping: true,
+    lineNumbers: true,
+    mode: 'text/x-sql',
+    readOnly: true
```

Comment:
Try readOnly: "nocursor" ? Currently it shows a cursor and just doesn't react to input. And/or add some custom CSS to make it look a bit different, i.e. light gray background?

Consider adding a tooltip on the "Last Query" button to explain what this is and how it differs from the SQL tab, or add this explanation as a comment prepended to the content.

It would be awesome if it would highlight the bits that differ from the SQL tab input, but that sounds difficult.

https://codemirror.net/doc/manual.html#config